### PR TITLE
ccl: update to 1.12.

### PR DESCRIPTION
**Pull request by rc-05**
Clozure-CL updated to latest [release](https://github.com/Clozure/ccl/releases).

### CODE REVIEW

**Comment by rc-05** [comment]
Pushed commit with new verified GPG key.

File: srcpkgs/ccl/template [hunk 1]
```diff
@@ -1,26 +1,31 @@
...
+python_version=3
 
 case $XBPS_MACHINE in
 	x86_64*) _arch=64 ;;
 	i686*) broken="Build hangs forever" ;;
 	*) _arch='' ;;
 esac
 
+pre_build() {
+	wget https://github.com/Clozure/ccl/releases/download/v${version}/linuxx86.tar.gz
```
Comment (Chocimier) [inline srcpkgs/ccl/template:25]: Add this to distfiles instead.

**Comment by Chocimier** [comment]
Otherwise tested, works.

File: srcpkgs/ccl/template (hunk 1 above)
Comment (rc-05) [reply srcpkgs/ccl/template:25]: Added but it can't find the extracted files.

Comment (Chocimier) [reply srcpkgs/ccl/template:25]: You need to move it into proper place, see srcpkgs/cgit/template .

**Comment by rc-05** [comment]
Thanks for the suggestion! I finally managed to make xbps-src move the needed files correctly without any sort of hack.

**Comment by Chocimier** [comment]
> I finally managed to make xbps-src move the needed files correctly without any sort of hack.

They aren't cleaned, however. xbps-src removes wrksrc after build. Here, when two archives are extracted, files from second are outside of wrksrc, so they aren't cleaned. This can be fixed by setting `create_wrksrc=yes` and following post-extract, so all files are extracted into wrksrc, then files from first tarball are moved one dir up into expected place. (I missed that second tarball breaks convention of having top-level dir.)
```
post_extract() {
	mv "ccl-${version}"/* .
}

```

Please also indent multiline variables by one space.

**Comment by rc-05** [comment]
Didn't know that honestly. Gotta take notes about that and change the template file as you suggested

**Pull request merged**